index.html

```diff
@@ -47,11 +47,173 @@ <h1 class="h-title">I’m ROBERTO. Glad to see you!</h1>
+    </p>
+
+    <ul>
+        <li class="tag">
+            <a href="">HTML</a>
+        </li>
+        <li class="tag">
+            <a href="" >CSS</a>
+        </li>
+        <li class="tag">
+            <a href="">JavaScript</a>
+        </li>
+    </ul>
+
+    <a href="#" class="button project">See Project</a>
+     </div>
+
+    
+
+</section>
+<section  class="card works">
+
+  <img src="img/Snapshoot2.png" alt="a_screentshot" class="pic">
+
+  <div class="text-card">
+      <h2 class="titleh2">Tonic</h2>
+      <ul class="canopy">
+        <li class="cano"><a href="#" class="can">CANOPY</a></li>
+        <li><a href="#">Back End Dev</a></li>
+        <li><a href="#">2015</a></li>
+      </ul>
+
+  <p class="text">
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+
+  <ul>
+      <li class="tag">
+          <a href="">HTML</a>
+      </li>
+      <li class="tag">
+          <a href="" >CSS</a>
+      </li>
+      <li class="tag">
+          <a href="">JavaScript</a>
+      </li>
+  </ul>
+
+  <a href="#" class="button project">See Project</a>
+   </div>
+
+  
+
+</section>
+<section  class="card works">
+
+  <img src="img/Snapshoot3.png" alt="a_screentshot" class="pic">
+
+  <div class="text-card">
+    <h2 class="titleh2"> Multi-Post Stories</h2>
+      <ul class="canopy">
+        <li class="cano"><a href="#" class="can">CANOPY</a></li>
+        <li><a href="#">Back End Dev</a></li>
+        <li><a href="#">2015</a></li>
+      </ul>
+
+  <p class="text">
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+
+  <ul>
+      <li class="tag">
+          <a href="">HTML</a>
+      </li>
+      <li class="tag">
+          <a href="" >CSS</a>
+      </li>
+      <li class="tag">
+          <a href="">JavaScript</a>
+      </li>
+  </ul>
+
+  <a href="#" class="button project">See Project</a>
+   </div>
+
+  
+
+</section>
+  <section class="myself-section">
+    <div class="title">
+        <h1 class="h-title">About Myself</h1>
+      </div>
+      <div class="parrafo">
+        <p>Hello I’m a software developer! I can help you build a product , feature or website.
+           Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+             don’t hesitate to contact me.
+        </p>
+      </div>
+
+
+<div class="connect">
+<img src="img/LET’S CONNECT.png" alt="connection image">
+</div>
+<div class="socialMedia">
+<ul class="ul">
+  <li><a href=""><img src="img/Vector.png" alt="" class="icon first"></a></li>
+  <li><a href=""><img src="img/Vector (1).png" alt="" class="icon"></a></li>
+  <li><a href=""><img src="img/Vector (2).png" alt="" class="icon icon-3"></a></li>
+  <li><a href=""><img src="img/Vector (3).png" alt="" class="icon"></a></li>
+  <li><a href=""><img src="img/Vector (4).png" alt="" class="icon"></a></li>
+</ul>
+<a href="#" class="last-button">See Project</a>
```

Comment:
- [ ] For the `About Section`, please put the right text content in your button. It should be `Get my resume` and not `See Project.` :+1: 
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/236204177-15af08af-1d99-45cb-9330-38b5c448e212.png)